OPAL/br/src/main/scala/org/opalj/br/analyses/cg/EntryPointFinder.scala

```diff
@@ -347,4 +347,46 @@ object AllEntryPointsFinder extends EntryPointFinder {
             project.allProjectClassFiles.flatMap(_.methodsWithBody)
         else project.allMethodsWithBody
     }
+}
+
+/**
+ * The AndroidEntryPointFinder considers specific methods of app components as entry points.
+ * It does not work for androidx
+ *
+ * @author Tom Nikisch
+ */
+object AndroidEntryPointsFinder extends EntryPointFinder {
+
+    val activityEPS: List[String] = List("onCreate", "onRestart", "onStart", "onResume",
```

Comment:
Avoid the abbreviation EPS for entry points, as in the context of OPAL, this is used extensively to mean Entity/Property-State.